### PR TITLE
Search hotfix release (11.2.1)

### PR DESCRIPTION
**Pull request by tjprescott**
Fixes https://github.com/Azure/azure-sdk-for-python/issues/22015

Since the next GA release for Search has been pushed out, and manually fixing the docs is not feasible, a hotfix release is the only viable solution.

### CODE REVIEW

File: sdk/search/azure-search-documents/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 11.2.1 (2022-01-5)
+
+Minor updates.
```
Comment (xiangyan99) [inline sdk/search/azure-search-documents/CHANGELOG.md:5]: **Other fixes**?

**Review by xiangyan99 — APPROVED** [review]

**Comment by xiangyan99** [comment]
Why we need .certificate/dotnet-devcert.pem ?

**Comment by xiangyan99** [comment]
> Why we need .certificate/dotnet-devcert.pem ?

And is there a security concern to check in cert?

**Review by xiangyan99 — CHANGES_REQUESTED** [review]
Please double check if checking in cert is a security concern.

**Comment by tjprescott** [comment]
@xiangyan99 the dev_cert is a dummy cert for test-proxy stuff so it's not a security concern. But, at the same time, it should NOT be checked in.

**Review by xiangyan99 — APPROVED** [review]

**Comment by tjprescott** [comment]
@scbedd I am trying to follow the guidance here (https://azure.github.io/azure-sdk/policies_repobranching.html#hotfix-branches) but am blocked by persistent dependency conflicts and issues. Your assistance would be greatly appreciated!

**Comment by scbedd** [comment]
Search CI is green. Core CI is failing due to changes in opentelemetry dependencies. Others are due to similar issues. GIven that this branch needs to release azure-search-documents only, we don't care about failing builds for the others. Overriding.

**Pull request merged**